features/ei-analytics-feature/org.wso2.analytics.solutions.ei.analytics.feature/src/main/resources/dashboards/ei-anlaytics.json

```diff
@@ -3410,7 +3401,7 @@
                                         "defaultValue": "admin"
                                     }
                                 ],
-                                "query": "from MediatorStatAgg on true AND componentId\u003d\u003d\u0027true\u0027 within 331237800000L, 394309800000L per \u0027minutes\u0027 select sum(noOfInvocation) as Count, sum(faultCount) as Fault, sum(noOfInvocation - faultCount) as Success, AGG_TIMESTAMP as Time group by AGG_TIMESTAMP"
+                                "query": "from MediatorStatAgg on true AND componentName\u003d\u003d\u0027test\u0027 within 331237800000L, 394309800000L per \u0027minutes\u0027 select minDuration as Minimum, maxDuration as Maximum, (totalDuration / noOfInvocation) as Average, AGG_TIMESTAMP as Time group by AGG_TIMESTAMP"
```

Comment:
Shouldn't the average be calculated in aggregation and not ON retrieval?

Reply:
Since it's needed to get the average by number of invocations, could not use siddhi average function for this.

Reply:
The incremental aggregation avg() can be used for the use case. Since the severity is minor, am approving now.
Please follow upon on https://github.com/wso2/analytics-solutions/issues/123 ASAP